Sources/SwiftSDKGenerator/Artifacts/DownloadableArtifacts.swift

```diff
@@ -45,74 +46,26 @@ enum ArtifactOS: Hashable {
 
 typealias CPUMapping = [Triple.CPU: String]
 
-/// SHA256 hashes of binary LLVM artifacts known to the generator.
-private let knownLLVMBinariesVersions: [ArtifactOS: [String: CPUMapping]] = [
-  .macOS: [
-    "15.0.7": [
-      Triple.CPU.arm64: "867c6afd41158c132ef05a8f1ddaecf476a26b91c85def8e124414f9a9ba188d",
```

Comment:
A side effect of using these hashes was that we also verified that the upstream tarballs hadn't changed from the known versions.   LLVM signs its binaries using GPG, so in future we could have an option to verify those signatures.   This would require the user to download and validate LLVM's public key.

Reply:
I agree, checking signatures is the right way to do it. But code on `main` that handled checksums was inconsistent and didn't warn users on checksums mismatch. It would only redownload new archives from remote URLs on a mismatch or if a checksum was absent. From that perspective, we didn't regress, and the caching behaviour is actually improved.